<launch>
  <!--node pkg="ir2425_group_08" type="node_a" name="node_a" output="screen"/-->
  <node pkg="ir2425_group_08" type="node_b" name="node_b" output="screen"/>
  <node pkg="ir2425_group_08" type="node_c" name="node_c" output="screen"/>
</launch>